lib/ret_web/controllers/api/v1/media_controller.ex

```diff
@@ -37,14 +37,20 @@ defmodule RetWeb.Api.V1.MediaController do
 
     case Ret.Storage.store(upload, content_type, access_token, promotion_token) do
       {:ok, uuid} ->
-        uri = Ret.Storage.uri_for(uuid, content_type)
+        origin_uri = Ret.Storage.uri_for(uuid, content_type)
+
+        raw_uri =
+          case content_type do
+            "application/pdf" -> gen_farspark_url(origin_uri, 0, "raw", "") |> URI.parse()
```

Comment:
This looks fine, but it might be an opportunity to refactor `gen_farspark_url`. The only other usage also uses `"raw", ""`, so maybe those arguments are redundant now.